frontend/src/components/pageComponents/Sessions/SessionList/GoalList/index.jsx

```
@@ -1,72 +1,73 @@
+          <li key={goal.name + index} className="mb-2 last:mb-0">
+  <button
+  className={`justify-between w-full p-4 flex items-center cursor-pointer transition-colors duration-300 rounded-lg shadow-md ${
+    expandedIndex === index
+      ? "bg-blue-100"
+      : "bg-gray-200 hover:bg-gray-300"
+  }`}
+  onClick={() =>
+    setExpandedIndex(expandedIndex === index ? null : index)
   }
+  aria-expanded={expandedIndex === index}
+>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)